Makefile

```diff
@@ -63,10 +49,10 @@ GO_BUILD := $(Q)go build
 build: bin/k8s bin/kuberpak
 
 bin/k8s:
-	$(GO_BUILD) -o $@ ./provisioner/k8s
+	CGO_ENABLED=0 go build -o $@ ./provisioner/k8s
```

Comment:
non-blocking: I'd think it would make more sense to just add this to the GO_BUILD variable instead. Thoughts?

Reply:
I guess it felt a bit awkward to use that go build variable as there's some parsing issues with ordering? We should tackle overhauling this Makefile at some point though as it's all over the place right now.